presto-phoenix/src/test/java/io/prestosql/plugin/phoenix/TestPhoenixDistributedQueries.java

```diff
@@ -75,7 +75,7 @@ protected TestTable createTableWithDefaultColumns()
     }
 
     @Override
-    public void testLargeIn()
+    public void testLargeIn(int valuesCount)
```

Comment:
We should be able to enable the test now for phoenix, provided that @sopel39 's capping works for phoenix too.